apps/wasm/mvp/utils.ts

```diff
@@ -7,3 +7,18 @@ import init from "./pkg/wasm"
 export async function initWasm() {
     await init(await fetch(wasmUrl))
 }
+
+export function downloadFile(path, filename = 'log') {
+    const anchor = document.createElement('a');
+    anchor.href = path;
+    anchor.download = filename;
+    anchor.style.display = 'none';
+    document.body.appendChild(anchor);
+    anchor.click();
+    document.body.removeChild(anchor);
+}
+
+export function textToPath(text, type = "text/plain") {
```

Comment:
are we exporting the logs as newline delimited json? if yes, shouldn't we use `application/json` as mime type?

Reply:
I'm not sure if I understand, but the newline delimited text is not a valid json

Reply:
usually log ingestion service support: https://jsonlines.org/ But i believe the use case is trivial as of now. So it should be fine